cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfOperationsCreator.java

```diff
@@ -10,9 +10,21 @@
 
 public class CfOperationsCreator {
 
+    /**
+     * Names of the environment variables that hold the username and the password value for the application.
+     */
     private static final String CF_CONTROL_USER = "CF_CONTROL_USER";
     private static final String CF_CONTROL_PASSWORD = "CF_CONTROL_PASSWORD";
 
+    /**
+     * Creates the cfOperations object, which is needed by our operations classes in the package. If the username and
```

Comment:
It looks a little bit much forr a constructor. 

I would recommend to add some documentation for the hole class. 
Variables explanations, I think you should put it to the constants. 

In the end, we will have a User-Doc, where we will describe, how to use it and specify our Programm.